libparsec/crates/platform_storage/tests/unit/workspace/mod.rs

```diff
@@ -3,6 +3,7 @@
 // `allow-unwrap-in-test` don't behave as expected, see:
 // https://github.com/rust-lang/rust-clippy/issues/11119
 #![allow(clippy::unwrap_used)]
+mod sync_pattern;
```

Comment:
Please stop doing this kind of move, while convenient at development time, it breaks the logic in the code hierarchy (why should we have everything in a `mod.rs`, then a specific sup-part in it own .rs file...)

Instead what you can do:
- solution1: follow the layout logic of the code
- solution2: properly re-split the code (typically asking on slack first if you are unsure how to split it)
- solution3: communicate on slack or during meeting about your pain-point with the codebase ;-)

Right now, the test file is ~1000lines long and you are adding ~100 more lines. I think it's just simpler to add your lines in the file, considering there is no much (if any) we are planning to add to the platform storage (so it's not worth refactoring the tests at the moment).

Reply:
done